docs - known issues (#23522)
metabase/metabase: docs/troubleshooting-guide/known-issues.md

---
title: How to find a known bug or limitation
---

# How to find a known bug or limitation

If you can't find or solve your problem using the [troubleshooting guides](./index.html), you may be running into an issue that we're still working on.

- [How to find a known bug](#how-to-find-a-known-bug) 
- [How to find a known product limitation](#how-to-find-a-known-product-limitation)

## How to find a known bug

> You may be encountering a bug if your work in Metabase is crashing, missing, or not being saved.

Reviewer: ```suggestion
If your work in Metabase is crashing, missing, or not being saved, you might have found a bug.
```

> If you think something is missing or behaving oddly, but you've never been able to do it before, try [searching for a product limitation](#how-to-find-a-known-product-limitation) instead.

Reviewer: ```suggestion
```

The bug v limitation links above cover this disambiguation.

1. Go to [Metabase Github issues page](https://github.com/metabase/metabase/issues).

2. Click on the **Label** dropdown and select `Type: Bug`.

> If you're encountering problems after your Metabase version was recently upgraded, you can add the label `.Regression`.

3. Choose least one value in the **Label** dropdown that matches the feature you're having trouble with.

> Add additional keywords to the search bar above the list of results. Hit Enter/Return to run the search.

> Click **Sort** and select the thumbs up emoji (👍) to view the most commonly encountered bugs.

5. If you find an existing bug that matches your problem, leave a thumbs up or comment to help the team prioritize it.

6. If you can't find your problem, and you're sure that it's not a [product limitation](#how-to-find-a-known-product-limitation), you can [create a new bug report](./bugs.html).

## How to find a known product limitation

> You may be encountering a product limitation if the functionality you're looking for doesn't exist.

Reviewer: ```suggestion
If the functionality you're looking for doesn't exist, upvote an existing feature request, or---if you can't find one---create a new feature request.
```

Is this what we want people to do in this case?

Author: The blurb at the top doesn't have instructions yet (I'll put the info about a feature request in the steps).

It's meant to help people figure out if something is missing because of a bug vs. a limitation... though only factor I could think of was that "if it's always been missing, and it didn't _suddenly_ disappear, it's probably a limitation and not a bug".

> If you're dealing with something that has suddenly disappeared or stopped working, try [searching for a bug](#how-to-find-a-known-bug) instead.

Reviewer: ```suggestion
```
Same as above.

Author: I put this kind of routing under each subheading in case people are linked directly to that subheading instead of starting at the top of the page. That's why it's a bit redundant 😅 

I ended up putting that info inside the steps instead!

1. Go to [Metabase Github issues page](https://github.com/metabase/metabase/issues).

2. Click on the **Label** dropdown and select `Type: New Feature`.

3. Choose least one value in the **Label** dropdown that matches the feature you're looking for.

> Add additional keywords to the search bar above the list of results. Hit Enter/Return to run the search.

> Click **Sort** and select the thumbs up emoji (👍) to view the most highly requested features.

> 5. If you find a feature request that matches what you want to do, leave a thumbs up or comment to help the team prioritize it.

Reviewer: Else create a new feature request? If so, link to the feature request page. 

And we might want to omit the reference to comments here. Comments like "Any updates on this?" will make @flamber frown. Comments that elaborate the problem that the feature would solve are helpful, The[ feature request page](https://www.metabase.com/docs/latest/troubleshooting-guide/requesting-new-features.html) already covers this, so we should just link to it).

Author: 🙏 I completely missed the feature request page!
